### PR TITLE
Feature git config

### PR DESCRIPTION
**Pull request by jakubkratina**
- `.gitconfig`
- `.gitignore_global`
- refers to: https://oakslab.atlassian.net/browse/EN-4

### CODE REVIEW

File: git/.gitignore_global [hunk 1]
```diff
@@ -0,0 +1,14 @@
+*.log
+nbproject
+.idea
+/node_modules
+bower_components
+.env
```
Comment (quibusus) [inline git/.gitignore_global]: I believe these should be in the project .gitignore

Everyone will have .gitignore, but only some will have gitignore_global - that might cause problems, e.g. someone accidentally pushing .env or node_modules

File: git/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+>
+
+## Highlights
+- git aliases
+- pull `rebase = true` as a default
+- colors
+- global `.gitignore`
+
+## Setup
+1. create `.gitconfig` and `.gitignore_global` files in your user's root directory (`cd ~` or `cd /Users/<username>`)
```
Comment (quibusus) [inline git/README.md]: How about using a separate file and include directive? Would make it easier to version/update the common config

`[include]
        path = /path/to/.gitconfig_ooo
`

File: git/.gitignore_global (hunk 1 above)
Comment (jakubkratina) [reply git/.gitignore_global]: It should definitely be inside project's `.gitignore`, but if you forget to add it there, `-gitignore_global` will catch it. It's like a fallback solution.

This does not replace project's `.gitignore`, it's only an addition (with some user specific files, for example if you use Netbeans, you ignore here `nbproject`, it shouldn’t be inside project's gitignore)

File: git/README.md (hunk 2 above)
Comment (jakubkratina) [reply git/README.md]: So you're suggesting to have this repository cloned as a separate project and then use links?

Comment (quibusus) [reply git/README.md]: I did not think of that, but that's a good idea!

**Comment by quibusus** [comment]
@jakubkratina let's go with the `[include] path = /path/to/.gitconfig_ooo` approach - I think it's less intrusive than direct .,gitconfig

thoughts?

**Comment by jakubkratina** [comment]
Where do you want to put the code? Into the existing `.gitconfig`?

If yes, I'll update the readme as:
- use include if you already have a config
- create one and put the content there if not (i know we have cases with no config at all)

**Comment by quibusus** [comment]
Very good ideas, let's do that!

File: git/README.md (hunk 2 above)
Comment (jakubkratina) [reply git/README.md]: Ok, it's there

Comment (jakubkratina) [reply git/README.md]: Included

**Comment by jakubkratina** [comment]
@quibusus Updated the readme.

**Pull request merged**